config/nrfconnect/chip-module/generate_factory_data.cmake

```diff
@@ -15,25 +15,29 @@
 #
 
 
-# Create a JSON file based on factory data given via kConfigs.
+# Create a .hex file in CBOR format based on factory data given via kConfigs.
```

Comment:
Yes, I know, but is it important from the cmake perspective? I didn't write about it, because after removing this split on two functions, we create .hex with this one and .json is used only internally as an intermediate product.

It looks a little bit different for the manual generation, the it's possible to generate only .json, but in cmake it will not be possible anymore.

Reply:
It is minor because I guess only a few people look at this description, but we don't remove that intermediate product and somebody may want to use it somehow.